### PR TITLE
Add pagination methods to RecordSet

### PR DESCRIPTION
**Pull request by gtimti**

### CODE REVIEW

**Comment by cheezy2022** [comment]
📦

File: lib/gap_intelligence/models/record_set.rb [hunk 1]
```diff
@@ -13,5 +13,23 @@ def initialize(records = nil, options = {})
...
+    def total_pages
+      pagination['total_pages']
+    end
+
+    def current_page
+      pagination['current_page']
+    end
+
+    def limit_value
+      pagination['total_count'] / total_pages if pagination['total_count'] && total_pages
```
Comment (chanko) [inline lib/gap_intelligence/models/record_set.rb]: Is there ever the possibility of total_pages being 0?

File: lib/gap_intelligence/models/record_set.rb [hunk 2]
```diff
@@ -13,5 +13,23 @@ def initialize(records = nil, options = {})
       @records = records || []
       @meta = options[:meta]
     end
+
+    def total_pages
+      pagination['total_pages']
```
Comment (chanko) [inline lib/gap_intelligence/models/record_set.rb]: Should this return a number? Potentially call ```.to_i```?

File: lib/gap_intelligence/models/record_set.rb [hunk 3]
```diff
@@ -13,5 +13,23 @@ def initialize(records = nil, options = {})
...
+    end
+
+    def current_page
+      pagination['current_page']
```
Comment (chanko) [inline lib/gap_intelligence/models/record_set.rb]: Should this return a number? Potentially call ```.to_i```?

Comment (gtimti) [inline lib/gap_intelligence/models/record_set.rb]: currently it returns Int values in pagination

File: lib/gap_intelligence/models/record_set.rb (hunk 1 above)
Comment (gtimti) [inline lib/gap_intelligence/models/record_set.rb]: thank you, fixed.

**Pull request merged**